summingbird-online/src/main/scala/com/twitter/summingbird/planner/OnlinePlan.scala

```diff
@@ -219,9 +232,12 @@ class OnlinePlan[P <: Platform[P], V](tail: Producer[P, V]) {
 }
 
 object OnlinePlan {
-  def apply[P <: Platform[P], T](tail: TailProducer[P, T]): Dag[P] = {
+  def apply[P <: Platform[P], T](tail: TailProducer[P, T]): Dag[P] =
+    apply(tail, scala.collection.immutable.Map.empty)
```

Comment:
Why not just have options as defaulting to `Map.empty`?  Are we worried about binary compatibility with this method?